phplib/FlashMessage.php

```diff
@@ -5,6 +5,12 @@ class FlashMessage {
   static $messages = [];
   static $hasErrors = false;
 
+  /**
+   *Adds messages to a messageQueue
+   * 
+   *@param string $message
+   *@param string $type success, warning, danger (default)
```

Comment:
Tehnic vorbind, și _info_ este un tip posibil (și util). Folosim exact [tipurile Bootstrap](https://getbootstrap.com/docs/3.3/css/#tables-contextual-classes).

Reply:
Mai mult pentru mine era doc-ul, că sunt străin deocamdată de parametrii posibili.